WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -275,5 +281,22 @@ async Task ReadyToSignTask(AliceClient aliceClient)
 				.Take(MaxInputsRegistrableByWallet)
 				.ToShuffled()
 				.ToImmutableList();
+
+		private ImmutableList<DateTimeOffset> CreateSchedule(DateTimeOffset startTime, TimeSpan timeFrame, int numberOfEvents) =>
```

Comment:
I would put this into a separate class (helper?).

Reply:
Helper classes are like garbage bins where developers put things that they don't know where really should be. I prefer to wait for the rest of the delays to be implemented and then it will become clear where it should be placed. I imagine this has to evolve to some kind of scheduder class but not sure yet.

Reply:
Helper classes can be tested though.

Reply:
i'm with Lucas here, although this is a simple function, it could even be just a static function (only depends on `Random` property) any sort of factoring out will require a lot of boilerplate and back and forth in review, i would prefer to refactor it later after we've figured out the bob client delays and we know what it's final form will be.

Reply:
ACK, we can move it if the class getting too big.